src/main/kotlin/sliv/tool/importer/ProjectParser.kt

```diff
@@ -2,42 +2,77 @@ package sliv.tool.importer
 
 import javafx.collections.FXCollections
 import javafx.scene.control.TreeItem
+import javafx.scene.image.Image
+import javafx.scene.image.ImageView
+import sliv.tool.parsers.planes.ImagePlanesParser.extractUIDs
+import sliv.tool.project.model.*
 import java.io.File
+import kotlin.io.path.Path
 
 object ProjectParser {
-    const val IMAGE_DIRECTORY_NAME = "images"
+    private const val IMAGE_DIRECTORY_NAME = "images"
+    private val images = FXCollections.observableHashMap<Long, String>()
+    private val layers = FXCollections.observableArrayList<ProjectLayer>()
+    private val landmarks = FXCollections.observableHashMap<Long, MutableList<LandmarkFile>>()
+    private val frames = FXCollections.observableArrayList<ProjectFrame>()
```

Comment:
this fields are used only in one method, so it is not right to save it as an object state. What if another directory will be parsed. Do you clear these collections?